fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/KeyValueCursorTest.java

```diff
@@ -419,4 +420,52 @@ public void emptyScanSplit() {
             return null;
         });
     }
+
+    @Test
+    public void buildWithoutRequiredProperties() {
+        try {
+            RecordCursor<KeyValue> kvCursor = KeyValueCursor.Builder.withSubspace(subspace)
+                                                .build();
+            // Program flow should not reach here.
+            assertTrue(false);
+        } catch (RecordCoreException ex) {
+            assertTrue(true);
+        }
+    }
+
+    @Test
+    public void buildWithoutScanProperties() {
+        fdb.run(context -> {
+            try {
+                RecordCursor<KeyValue> kvCursor = KeyValueCursor.Builder.withSubspace(subspace)
+                                                    .setContext(context)
+                                                    .build();
+                // Program flow should not reach here.
+                assertTrue(false);
+            } catch (RecordCoreException ex) {
+                assertTrue(true);
+            }
+
+            return null;
+        });
+    }
+
+    @Test
+    public void buildWitRequiredProperties() {
+        fdb.run(context -> {
+            try {
+                RecordCursor<KeyValue> kvCursor = KeyValueCursor.Builder.withSubspace(subspace)
+                                                    .setContext(context)
+                                                    .setScanProperties(ScanProperties.FORWARD_SCAN)
+                                                    .build();
+                assertTrue(true);
+            } catch (RecordCoreException ex) {
```

Comment:
I don't *think* there's an `assertDoesntThrow` assertion, though maybe I'm wrong. Maybe just build the key value cursor and then leave a comment that says that this is to validate that no exception is thrown? If one is thrown, then the test will fail anyway.

Or maybe this is fine.

Reply:
DONE.
I kept the try/catch. Should the function being tested, throw an exception other than 'RecordCoreException', this test would still fail. Please let me know if that is acceptable.

Reply:
Hm, I suppose it's fine.